README.rst

```diff
@@ -432,6 +432,13 @@ shows the different configuration options available:
         <or>
         platform: linux/arm64/v8 # an apple m1 architecture
 
+        # Multi-platform images
+        # Specify a list of platforms to build for, and buildrunner will build a manifest
+        # list image with the given platforms
```

Comment:
I think we need to make it clear that this uses the builders configured in build for these platforms, which may or may not be emulated.

Reply:
I agree this should be more clear. But I am not sure what you mean by `this uses the builders configured in build for these platforms`. Can you elaborate on that some more?

Reply:
builders configured in *buildx* is what I meant.

Reply:
Oh I see what you mean. Right now, it doesn't use any configured `buildx builders`, it just uses the default builder. In the future we may use different builders.

However, for now how about this for the comment?
        # For Multi-platform images please specify a list of platforms to build for
        # and buildrunner will build images with the given platforms.
        # Please note that these may be built using emulated architectures which
        # will be slower than native builds.

Reply:
Updated this, but let me know if you think it still needs work.